src/dataset-builder/DatasetBuilderUtils.ts

```diff
@@ -146,20 +146,20 @@ export const createSnapshotBuilderCountRequest = (cohort: Cohort[]): SnapshotBui
   return { cohorts: _.map(convertCohort, cohort) };
 };
 
-export const HighlightConceptName = ({ conceptName, searchFilter }): ReactElement => {
-  const startIndex = conceptName.toLowerCase().indexOf(searchFilter.toLowerCase());
+export const HighlightSearchText = ({ columnItem, searchFilter, style = {} }): ReactElement => {
```

Comment:
Do you think it would be possible to add types to the props here?
```suggestion
export const HighlightSearchText = (props: HighlightSearchTextProps): ReactElement => {

const { columnItem, searchFilter, style }
```